package.json

```diff
@@ -54,6 +53,7 @@
     "babel-preset-env": "^1.6.0",
     "babel-register": "^6.24.1",
     "codeclimate-test-reporter": "^0.5.0",
+    "cross-env": "^5.0.5",
```

Comment:
@Ardeshir81 Need to add missing `yarn.lock`

Reply:
oooo .... : )))
I added it using npm . Should have used yarn of course ....